internal/data/models.go

```diff
@@ -0,0 +1,17 @@
+package data
+
+import (
+	"errors"
+	"wallet-backend/internal/db"
+)
+
+type Models struct {
+}
+
+func NewModels(db db.DBConnectionPool) (*Models, error) {
+	if db == nil {
+		return nil, errors.New("DBConnectionPool must be initialized")
```

Comment:
Looks like `errors.New("DBConnectionPool must be initialized")` will be a common error for different models. Can we make it a global variable in the data package?